exist-core/src/main/java/org/exist/xquery/GeneralComparison.java

```diff
@@ -330,8 +330,7 @@ public NodeSet preSelect( Sequence contextSequence, boolean useContext ) throws
         
         // if the right hand sequence has more than one item, we need to merge them
         // into preselectResult
-        if (rightSeq.getItemCount() > 1)
-        	{preselectResult = new NewArrayNodeSet();}
+        if (rightSeq.getItemCount() > 1) {preselectResult = new NewArrayNodeSet();}
```

Comment:
formatting is not worse or better :-)

Reply:
Hmm I think IntelliJ did that one by itself - I don't like it!